test/env

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -e
+
+exec 1000> /dev/stdout
+BASH_XTRACEFD=1000
+echo "$0" "$@"
+set -x
+
+if [ -z "${MPV_MPRIS_TEST_PLUGIN+set}" ] ; then
+	export MPV_MPRIS_TEST_PLUGIN=../mpris.so
+fi
+
+if [ -z "$MPV_MPRIS_TEST_PLAY" ] ; then
+	export MPV_MPRIS_TEST_PLAY=/usr/share/sounds/freedesktop/stereo/alarm-clock-elapsed.oga
+fi
+
+if [ -z "$MPV_MPRIS_TEST_IPC" ] ; then
+	export MPV_MPRIS_TEST_IPC=.
+fi
+
+if [ -z "$MPV_MPRIS_TEST_LOG" ] ; then
+	export MPV_MPRIS_TEST_LOG=.
+fi
+
+exec \
+env \
+MPV_MPRIS_TEST_NAME="$1" \
+XDG_RUNTIME_DIR="$MPV_MPRIS_TEST_TMP" \
```

Comment:
This fails unless `MPV_MPRIS_TEST_TMP` is set to something, i.e. I get:
```
+ exec env MPV_MPRIS_TEST_NAME=./metadata XDG_RUNTIME_DIR= dbus-run-session -- xvfb-run -a ./metadata
dbus[32]: Unable to set up transient service directory: XDG_RUNTIME_DIR "" not available: No such file or directory
```
in an environment which has `XDG_RUNTIME_DIR` unset (which is not an issue in general).

Reply:
Potentially, something like:
```bash
if [ -n "${MPV_MPRIS_TEST_TMP}" ]; then
  export XDG_RUNTIME_DIR="${MPV_MPRIS_TEST_TMP}";
fi
```
could accomodate the use case to allow for `XDG_RUNTIME_DIR` do stay unset if wanted, and still be overridable.

Reply:
Thanks, will fix that and I think make the variable a bit less generic.

The thinking with setting `XDG_RUNTIME_DIR` is to avoid polluting the system dirs as much as possible.

Reply:
> The thinking with setting XDG_RUNTIME_DIR is to avoid polluting the system dirs as much as possible.

That's a good goal. While of course it won't matter in sandboxed test environments, users may also want to run this manually. It might be helpful to also mention the variable in `README.md` to further encourage use of it ;-).